client-js/app/scripts/app.ts

```diff
@@ -4,6 +4,8 @@ import page = require("page");
 
 import registry = require("./registry");
 import datavault = require("./datavault");
+import dumb_server = require("./dumb_server");
+import client_rpc = require("./client_rpc");
```

Comment:
The convention in js is to use `camelCase` for variable names. We should stick with that. Files named with underscores are ok, but I think the convention in the js world is to use `hyphen-separated-names` for files.